Description: remove SONAME
 libnss_wrapper is a preloadable library.
 We do not support directly linking to it, so it doesn't need the SONAME.
 Let's remove it.
Author: Jakub Wilk <jwilk@debian.org>
Last-Update: 2014-06-01

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -17,10 +17,7 @@ target_link_libraries(nss_wrapper ${NWRA
 set_target_properties(
   nss_wrapper
     PROPERTIES
-      VERSION
-        ${LIBRARY_VERSION}
-      SOVERSION
-        ${LIBRARY_SOVERSION}
+      NO_SONAME ON
 )
 
 install(TARGETS nss_wrapper
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,11 +23,11 @@ set(APPLICATION_NAME ${PROJECT_NAME})
 #     Increment MINOR. Set PATCH to 0
 #   If the source code was changed, but there were no interface changes:
 #     Increment PATCH.
-set(LIBRARY_VERSION_MAJOR 0)
-set(LIBRARY_VERSION_MINOR 3)
-set(LIBRARY_VERSION_PATCH 1)
-set(LIBRARY_VERSION "${LIBRARY_VERSION_MAJOR}.${LIBRARY_VERSION_MINOR}.${LIBRARY_VERSION_PATCH}")
-set(LIBRARY_SOVERSION ${LIBRARY_VERSION_MAJOR})
+#set(LIBRARY_VERSION_MAJOR 0)
+#set(LIBRARY_VERSION_MINOR 3)
+#set(LIBRARY_VERSION_PATCH 1)
+#set(LIBRARY_VERSION "${LIBRARY_VERSION_MAJOR}.${LIBRARY_VERSION_MINOR}.${LIBRARY_VERSION_PATCH}")
+#set(LIBRARY_SOVERSION ${LIBRARY_VERSION_MAJOR})
 
 # add definitions
 include(DefineCMakeDefaults)
